components/servo/lib.rs

```diff
@@ -998,6 +999,28 @@ impl Servo {
                     webview.delegate().show_form_control(webview, form_control);
                 }
             },
+            EmbedderMsg::GetWindowRect(webview_id, response_sender) => {
+                let window_rect = if let Some(webview) = self.get_webview_handle(webview_id) {
+                    let dpi = webview.dpi();
+                    let screen = webview.delegate().screen_geometry(webview).unwrap();
+                    let pos = screen.window_offset.to_f32();
```

Comment:
Nit (sorry, I like to avoid generic abbreviations like `pos` as it makes the following code easier to understand):

```suggestion
                    let window_offset = screen.window_offset.to_f32();
```